### DISCUSSION TITLE
New Build of Custom Kernel Exynos 9820

### DISCUSSION THREAD

**Discussion by ghost** [community/community#146277] [Actions]
### Select Topic Area

Question

### Body

Hello:
This is my first time in this area and I'm a very new user. I'm trying to customize a new build for my kernel. I'm with this repository
https://github.com/Star-Seven/samsung-exynos9820

It's very clear how to update the code but my hell starts after clicking the button "Comitt Changes". I receive no feedback nor email notifications. I have waited a couple of hours and nothing.

In a couple of weeks ago I was with the original repository and I always received the same error 
"Error: This request has been automatically failed because it uses a deprecated version of `actions/upload-artifact: v2`. Learn more: https://github.blog/changelog/2024-02-13-deprecation-notice-v1-and-v2-of-the-artifact-actions/"
After reading a little I switched to this new repository because I understood that this repository was in an old version and I could not
do anything on my own.
Basically I would like to know how I can find out what's is going on with this new repository that I don't receive message about
the new buidl
Thanks for any help
Ozz
